blackbox/docs/protocols/postgres.txt

```diff
@@ -38,15 +38,34 @@ CrateDB emulates PostgreSQL server version ``9.5``
 Start-Up
 --------
 
-The connection handshake is implemented without authentication support. CrateDB
-will always respond with ``AuthenticationOK`` and any SSLRequests are answered
-with ``N`` to indicate that it does not support SSL.
+SSL Support
+...........
+
+SSL support is only available in the `Enterprise Edition`_. If enterprise is
+disabled, any ``SSLRequests`` are always answered with ``N``, indicating to a
```

Comment:
this phrase doesn't sound correct, i checked with @Autophagy. as a suggestion you can use : `If the Enterprise Edition is disabled, all SLLRequests are answered with N, indicating to the client that SSL is not available.`